lib/galaxy/config/sample/object_store_conf.xml.sample

```diff
@@ -52,6 +52,7 @@
             <cache path="database/object_store_cache_irods" size="1000" />
             <extra_dir type="job_work" path="database/job_working_directory_irods"/>
             <extra_dir type="temp" path="database/tmp_irods"/>
+            <icommands put_cutoff="100000000" num_threads="1"/>
```

Comment:
Files larger than put_cutoff are pushed to irods server via icommands. num_threads is 1 for now. But I want to have the flexibility to change it if need be.